app/models/question.rb

```diff
@@ -1,3 +1,12 @@
 class Question < ActiveRecord::Base
   has_many :answers
+
+
+    def display_resolved
```

Comment:
I noticed you created this method but are you using it in the views. I would leave this outside of the model though. Remember that in the MVC pattern, models should be independent from the views. This logic is a bit specific to the view and is creating a dependency. Think about if we needed to add translations to the view, or implement it in a mobile device.